dashboard/config/routes.rb

```diff
@@ -1017,6 +1017,11 @@
     get 'project_commits/get_token', to: 'project_commits#get_token'
     get 'project_commits/:channel_id', to: 'project_commits#project_commits'
 
+    # partial ports of legacy v3 APIs
+    get '/v3/channels/:channel_id/abuse', to: 'report_abuse#show_abuse'
+    delete '/v3/channels/:channel_id/abuse', to: 'report_abuse#destroy_abuse'
+    post '/v3/channels/:channel_id/abuse/delete', to: 'report_abuse#destroy_abuse'
```

Comment:
I suspect we can get rid of the `post` route. IIRC this was for compatibility with very old versions of IE.